### PR TITLE
rename filename and update copyright

### PR DESCRIPTION
**Pull request by luizcmarin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,94 @@
...
+    </a> 
+    <a href="https://coveralls.io/github/tempestphp/tempest-framework?branch=main" target="_blank">
+        <img alt="Coverage Status" src="https://coveralls.io/repos/github/tempestphp/tempest-framework/badge.svg?branch=main">
+    </a>
+</p>
+
+[TempestPHP](https://github.com/tempestphp/tempest-framework) is a PHP MVC micro-framework that gets out of your way. Its design philosophy is that developers should write as little framework-related code as possible, so that they can focus on application code instead. Our primary goal is to provide a structured framework that enables PHP users at all levels to rapidly develop robust web applications, without any loss to flexibility.
+
+
+## Installing TempestPHP via Composer
```
Comment (brendt) [inline README.md:19]: I'm a bit confused, why four ways of writing Tempest?

- TEMPEST
- TempestPHP
- tempestphp
- Tempest

The framework's name is "Tempest"

File: README.md [hunk 2]
```diff
@@ -1,25 +1,94 @@
...
+
+Installing TempestPHP into a project means that it will copy one or two files into that project:
+
+- `public/index.php` — the web application entry point
+- `tempest` – the console application entry point
+
+You can choose which files you want to install, and you can always rerun the `install` command at a later point in time.
+
+
+## Running Tests
```
Comment (brendt) [inline README.md:62]: This section is wrong, people need to run `composer qa` and not have PHPUnit installed globally as it's a local dependency

File: README.md [hunk 3]
```diff
@@ -1,25 +1,94 @@
...
+Assuming you have PHPUnit installed system wide using one of the methods stated [here](https://phpunit.de/manual/current/en/installation.html), you can run the tests for tempestphp by doing the following:
+
+1. Copy `phpunit.xml.dist` to `phpunit.xml`.
+2. Add the relevant database credentials to your `phpunit.xml` if you want to run tests against a non-SQLite datasource.
+3. Run `phpunit`.
+
+
+## Some Handy Links
+
+* [TempestPHP](https://github.com/tempestphp/tempest-framework) - The rapid development PHP framework.
```
Comment (brendt) [inline README.md:73]: This tagline is wrong, it's CakePHP's tagline.

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    <a href="LICENSE" target="_blank">
+        <img alt="Software License" src="https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat-square">
+    </a> 
+    <a href="https://coveralls.io/github/tempestphp/tempest-framework?branch=main" target="_blank">
+        <img alt="Coverage Status" src="https://coveralls.io/repos/github/tempestphp/tempest-framework/badge.svg?branch=main">
+    </a>
+</p>
+
+
+## Contributor Code of Conduct
```
Comment (brendt) [inline CODE_OF_CONDUCT.md:17]: Let's not add a CoC for now. This requires a more thorough discussion amongst core members, and not a copy/paste from other repositories.

**Comment by brendt** [comment]
I notice this is mostly a copy/paste from Cake's readme and CoC. I don't feel comfortable merging this in as a copy. I've left some comments, but I feel that it needs more careful thought that copying over another framework's Readme.

**Comment by luizcmarin** [comment]
taking advantage of the opportunity..... how do I make each PR individual as you wanted?

**Comment by luizcmarin** [comment]
I'm trying to collaborate on the documentation. cakePHP and others have a good logical structure. As I don't know anything about this project, I'm studying it in parts. And some things copied from other places can be adapted to the reality of the project, gaining its personality as it grows. See my suggestions as they are: a draft that will undergo many mutations as it gains body, muscles and personality. You draw your work of art, and I'm trying to decipher it...

**Comment by luizcmarin** [comment]
in other words: "we don't need you."
I've done so much in the last few days based on tempest, from documentation to validation routines they are doing and tests/corrections in the codes that I was very excited, but I didn't imagine being slapped in the face like that. I thought there would be more dialogue. Well done to me.

**Pull request closed without merging**